bot.py

```diff
@@ -1,107 +1,126 @@
 import asyncio
 import logging
 import sys
-import traceback
+import typing
+from datetime import datetime
+from typing import Annotated, Optional, Literal
+from zoneinfo import ZoneInfo
 
 import discord
+from discord.ext import commands
 from dynaconf import ValidationError
 
+import constants
 import lib
-from messages import SubmitMessage, GetBestTimesMessage
-from database import Database
 from config import settings
-import constants
+from database import Database
+from error_handler import ErrorHandlerCog
 
 logger = logging.getLogger(__name__)
 
 
-class MyBot(discord.Client):
+class MyBot(commands.Bot):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.queue = asyncio.Queue()
         self.db = Database().get()
 
+    async def setup_hook(self):
+        await asyncio.gather(
+            bot.add_cog(Commands(bot)),
+            bot.add_cog(ErrorHandlerCog(bot))
+        )
+
     async def on_ready(self):
         print("Logged in as", self.user)
         while True:
             try:
                 submit_msg = await self.queue.get()
                 logger.info("Going to process submission from queue: %s", submit_msg)
-                await lib.benchmark(submit_msg)
+                await lib.benchmark(*submit_msg)
                 self.queue.task_done()
             except Exception:
                 logger.exception("Error while processing submission.")
 
-    async def handle_best(self, msg):
+
+def today():
+    dt = datetime.now(tz=ZoneInfo("America/New_York"))
+    return min(dt.day, constants.MAX_DAY)
```

Comment:
Is there a reason you didn't import `today()` from `lib.py`?

(Yes, I see the bug around `constants.MAX_DAY` vs. hard-coding 25, that should be fixed anyway)

Reply:
did not know that there was this function in lib.py, i was just copying the function that existed in messages.py. will fix